idp-core/src/main/java/io/mosip/idp/core/dto/WalletBindingRequest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.idp.core.dto;
+
+
+import java.util.Map;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+
+import io.mosip.idp.core.util.ErrorConstants;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class WalletBindingRequest {
+	
+	@NotBlank(message = ErrorConstants.INVALID_TRANSACTION_ID)
+    private String transactionId;
+
+    @NotBlank(message = ErrorConstants.INVALID_IDENTIFIER)
+    private String individualId;
+
+	@NotBlank(message = ErrorConstants.INVALID_AUTH_CHALLENGE)
+    private AuthChallenge authChallenge;
```

Comment:
Later we may want to support multi factor auth for binding, can we allow that like we allowed in the authenticate API call ?

Reply:
Fixed